src/components/Product/ListProduct.js

```diff
@@ -17,13 +22,23 @@ const ListProduct = () => {
               </tr>
             </thead>
             <tbody>
-              {data.map((value, key) => (
-                <tr key={key}>
-                  <td>{value.name}</td>
-                  <td>{value.date} </td>
-                  <td>{value.lastDate}</td>
-                </tr>
-              ))}
+              {data.map((value, key) => {
+                const differenceDays = differenceInDays(
+                  value.lastDate.seconds,
+                  currentDateSeconds,
+                );
```

Comment:
This is not working as expected, my guess is that `value.lastDate.seconds` is returning `undefined`.

What if you store the date when the product was last added, in seconds instead?

Reply:
the issue was fixed